daily-challenges/2024-10/22/hunghoang3110.spec.ts

```
@@ -0,0 +1,54 @@
+import {expect, test} from '@playwright/test';
+import path from "path";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)